packages/EasyDecision/src/Bridge/Laravel/EasyDecisionServiceProvider.php

```diff
@@ -70,6 +80,10 @@ function (): ExpressionLanguageFactoryInterface {
             }
         );
 
+        $this->app->singleton(MappingProviderInterface::class, static function (): MappingProviderInterface {
```

Comment:
Why do we need to duplicate it here? We've registered this interface in the `register` method already.